### ISSUE TITLE
Trying to build unit tests in GA

### ISSUE THREAD

**Issue by bpbond** [JGCRI/hector#452]
My revised makefiles in #448 build Hector and hector-unit-tests locally just fine. They also build hector in GitHub Actions. But for hector-unit-tests, a no target to build error occurs.

Make 3.81 locally, 4.2.1 on GA.

```
ar: `u' modifier ignored since `D' is the default (see `U')
ar: creating libhector.a
make -C Testing hector-unit-tests
make[2]: *** No rule to make target 'hector-unit-tests'.  Stop.
make[2]: Entering directory '/home/runner/work/hector/hector/src/Testing'
make[2]: Leaving directory '/home/runner/work/hector/hector/src/Testing'
make[1]: Leaving directory '/home/runner/work/hector/hector/src'
make[1]: *** [makefile.standalone:82: testing] Error 2
```

It has the correct $OBJS:
>OBJS: test_logger.o testing_main.o test_hreader.o test_tseries.o test_fluxpool.o test_ini_to_core_reader.o test_inih.o

To double-check if version difference is the issue, I installed GNU Make 4.3 onto my machine; everything builds fine.

**Comment by bpbond** [comment]
Oh. My. God.
<img width="708" alt="Screen Shot 2021-07-08 at 8 01 27 PM" src="https://user-images.githubusercontent.com/1956468/125004907-86cfa180-e028-11eb-9c60-606bb7e3a8e5.png">

**Comment by bpbond** [comment]
This was ridiculously hard to figure out. Just for the record: only a single `Testing/` subdirectory existed (appeared, even when doing `ls -a`) on my local machine, but on Ubuntu, it magically split into `testing/` and `Testing/`. 🤦 

Fixed and merged in #451
